run-experiment.sh

```diff
@@ -42,11 +68,23 @@ dump_criu_app() {
 	echo "App PID [$APP_PID]"
 	ps aux | grep java
 
-	echo "Warming $APP_DIR App"
-	while [[ "$(curl --header 'X-Warm-Request: true' -s -o /dev/null -w ''%{http_code}'' http://$HTTP_SERVER_ADDRESS/ping)" != "200" ]]; 
-	    do sleep 1;
-	done
-	curl http://$HTTP_SERVER_ADDRESS/gc
+	if [ -n "$WARM_REQ" ];
+	then 
+		echo "Warming $APP_DIR App"
+		while [[ "$(curl --header 'X-Warm-Request: true' -s -o /dev/null -w ''%{http_code}'' http://$HTTP_SERVER_ADDRESS/ping)" != "200" ]]; 
```

Comment:
super strange this ping. Could youy please confirm?

Reply:
yep, now I see that was a weird design decision, but the endpoint _**ping**_ call the function handle. I plan to remove this endpoint in the future!